lib/plugins/healthcheck/commonHealthcheckPlugin.ts

```diff
@@ -9,6 +9,7 @@ export interface CommonHealthcheckPluginOptions {
   logLevel?: 'fatal' | 'error' | 'warn' | 'info' | 'debug' | 'trace' | 'silent'
   healthChecks: readonly HealthCheck[]
   infoProviders?: readonly InfoProvider[]
+  isRootRouteDisabled?: boolean
```

Comment:
generally it's recommended to have parameters in positive, e. g. isRootRouteEnabled. but it can default to `true`, which it should

Reply:
Agree, changed!